app/views/guides/_form.html.erb

```diff
@@ -77,3 +77,35 @@
     </fieldset>
   <% end %>
 </div>
+
+<div class='well comments'>
```

Comment:
I don't think this should live in this file. Perhaps you can have it in edit.html.erb?